tests/test_api/test_worker.py

```diff
@@ -210,7 +210,13 @@ def testApiWorker_ExtractMetaDataID(filesDir, mockXml):
         '</mmd:mmd>'
     )%("a1ddaf0f-cae0-4a15-9b37-3468e9cb1a2b")
 
-    namespaced_UUID_bad = (
+    namespaced_UUID_empty_return = (
```

Comment:
variable name seems misleading. It's the missing namespace, which is the problem, isn't it?

Reply:
":a1ddaf0f-cae0-4a15-9b37-3468e9cb1a2b".split(":") returns two strings: "" and "a1ddaf0f-cae0-4a15-9b37-3468e9cb1a2b"

Reply:
This test invokes empty return for namespace.